backend/templates/base.html

```diff
@@ -0,0 +1,41 @@
+{% load cms_tags menu_tags sekizai_tags %}
+<DOCTYPE html>
+<html lang = "{{ LANGUAGE_CODE}}">
```

Comment:
Please remove whitespace around `=`. It should read `lang="{{ LANGUAGE_CODE }}"` to be consistent with attributes elsewhere in the HTML.